### PR TITLE
Update sftp script and remove remote files after download

### PR DESCRIPTION
**Pull request by moh-moola**

### CODE REVIEW

File: scripts/SFTPImport.sh [hunk 1]
```diff
@@ -35,7 +35,13 @@ do
...
 
+# get the files from remote
 sshpass -p ${PASSWORD:? $P_ERROR} sftp -a ${USERNAME:? $RU_ERROR}@${REMOTE_HOST:? $RH_ERROR}:${FILES:- *} ${DIR:? $D_ERROR}
 
 # upload downloaded to S3
 s3cmd sync --secret_key=${SECRET_KEY} --access_key=${ACCESS_KEY} ${DIR:? $D_ERROR}/${FILES:- *} s3://${BUCKET:? S3_ERROR}
+
+# remove the files from remote
+sshpass -p ${PASSWORD:? $P_ERROR} sftp ${USERNAME:? $RU_ERROR}@${REMOTE_HOST:? $RH_ERROR}
+rm -rf ${FILES:- *}
```
Comment (KaitCrawford) [inline scripts/SFTPImport.sh]: I'm a bit unfamiliar with bash syntax, what does `:- *` do on this line?

Comment (moh-moola) [reply scripts/SFTPImport.sh]: default to wild card if var is null/empty

Comment (moh-moola) [reply scripts/SFTPImport.sh]: :face_with_head_bandage: I guess I didn't think that one through... I'll add some safety check

Comment (moh-moola) [reply scripts/SFTPImport.sh]: @KaitCrawford I can happily confirm that `rm` already has a safety mechanism in place for this

**Pull request merged**